src/clusterfuzz/_internal/bot/tasks/commands.py

```diff
@@ -43,108 +43,38 @@
-  return UTask(task_module)
-
-
-class UTask(BaseTask):
-  """Represents an untrusted task. Executes it entirely locally."""
-
-  def execute(self, task_argument, job_type, uworker_env):
-    """Executes a utask locally."""
-    preprocess_result = utasks.tworker_preprocess(self.module, task_argument,
-                                                  job_type, uworker_env)
 
-    if preprocess_result is None:
-      return
+def initialize_command_map(command_map):
+  assert set(COMMAND_MODULE_MAP.keys()) == set(task_types.COMMAND_TYPES.keys())
+  for task_name, task_callable in task_types.COMMAND_TYPES.items():
+    task_module = COMMAND_MODULE_MAP[task_name]
+    task = task_callable(task_module)
+    command_map[task_name] = task
 
-    input_download_url, output_download_url = preprocess_result
-    utasks.uworker_main(self.module, input_download_url)
-    # Postprocessing is done on a different instance.
-    del output_download_url  # Don't need this, it's for a different instance.
-    logs.log('Utask local: done with preprocess and main.')
 
-
-class UTaskLocalExecutor(BaseTask):
-  """Represents an untrusted task. Executes it entirely locally and in
-  memory."""
-
-  def execute(self, task_argument, job_type, uworker_env):
-    """Executes a utask locally in-memory."""
-    uworker_input = utasks.tworker_preprocess_no_io(self.module, task_argument,
-                                                    job_type, uworker_env)
-    if uworker_input is None:
-      return
-    uworker_output = utasks.uworker_main_no_io(self.module, uworker_input)
-    if uworker_output is None:
-      return
-    utasks.tworker_postprocess_no_io(self.module, uworker_output, uworker_input)
-    logs.log('Utask local: done.')
-
-
-class PostprocessTask(BaseTask):
-  """Represents postprocessing of an untrusted task."""
-
-  def __init__(self, module='none'):
-    # We don't need a module, postprocess isn't a real task, it's one part of
-    # many different tasks.
-    super().__init__(module)
-
-  def execute(self, task_argument, job_type, uworker_env):
-    """Executes postprocessing of a utask."""
-    # These values are false for now.
-    del job_type
-    del uworker_env
-    input_path = task_argument
-    utasks.tworker_postprocess(input_path)
-
-
-COMMAND_MAP = {
-    # TODO(metzman): Change analyze task away from in-memory.
-    'analyze': utask_factory(analyze_task),
-    'blame': TrustedTask(blame_task),
-    'corpus_pruning': utask_factory(corpus_pruning_task),
-    'fuzz': utask_factory(fuzz_task),
-    'impact': TrustedTask(impact_task),
-    'minimize': utask_factory(minimize_task),
-    'progression': utask_factory(progression_task),
-    'regression': utask_factory(regression_task),
-    'symbolize': TrustedTask(symbolize_task),
-    'unpack': TrustedTask(unpack_task),
-    'postprocess': PostprocessTask(),
-    'upload_reports': TrustedTask(upload_reports_task),
-    'variant': utask_factory(variant_task),
-}
+def get_command_map():
```

Comment:
Is there any reason to compute this lazily? Can we just do this on module import?